tests/main.test.ts

```diff
@@ -19,15 +16,13 @@ const htmlUrl = "https://github.com/ubiquibot/automated-merging/pull/1";
 const actionsGithubPackage = "@actions/github";
 const issueParams = { owner: "ubiquibot", repo: "automated-merging", issue_number: 1 };
 const workflow = "workflow";
+const githubHelpersPath = "../src/helpers/github";
+const monitor = "ubiquibot/automated-merging";
```

Comment:
If we migrate to @ubiquity-os organization then we might need to change these.

Reply:
These are used only for testing.